### ISSUE TITLE
Subcommand aliases are broken

### ISSUE THREAD

**Issue by ylectric** [facebookincubator/python-nubia#43]
After applying the patch from [this gist:](https://gist.github.com/ylectric/8b6c1fa6b4ae2ca0bf2bbdce23a8af14)
```
~/github/python-nubia master*
.venv ❯ env PYTHONPATH=(pwd) python example/nubia_example.py super-command do
usage: nubia_example.py super-command [-h] [--config CONFIG] [--verbose]
                                      [--stderr] [--shared INT]
                                      [subcommand] ...
nubia_example.py super-command: error: argument [subcommand]: invalid choice: 'do' (choose from 'do-stuff', 'print-name')
```

**Comment by marimar96** [comment]
**I will give it a try to this** !

**Comment by ylectric** [comment]
@marimar96, that's great! Feel free to ask any relevant questions.

**Comment by ylectric** [comment]
Fixed by #46
